src/nurse/views.py

```diff
@@ -24,16 +25,20 @@
 
 class SendEmailToDoctorView(APIView):
     def post(self, request, pk):
-        prescription = get_object_or_404(Prescription, id=pk)
+        serializer = PrescriptionEmailSerializer(data=request.data)
+        try:
+            serializer.is_valid(raise_exception=True)
+        except serializers.ValidationError as e:
+            return Response(e.detail, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
Why not keeping things simple? We didn't have this try/catch before, I don't get why it is added?
Default behavior of is_valid(raise_exception=True) is to return a 400 already.
Rule of thumb try/catch need to be demonstrated with a test.